src/index.css

```diff
@@ -137,7 +133,7 @@ input:focus {
 /* Landing Page */ 
 
 .landing {
-  margin-bottom: 150px;
+  margin-bottom: 50px;
```

Comment:
I think it would be nicer to have this buffer (`150px`) underneath the email box. Gives more breathing room to the home/landing section.